Inc/Model.h

```diff
@@ -53,10 +53,14 @@ namespace DirectX
         uint32_t                                                vertexOffset;
         uint32_t                                                vertexStride;
         uint32_t                                                vertexCount;
+        uint32_t                                                indexBufferSize;
```

Comment:
For simplicity, I require these values both be populated if you are using either dynamic or static VB/IB. It's a breaking change for custom loaders and renderers, but an easy one to resolve.